opentech/apply/review/tests/factories/models.py

```diff
@@ -40,3 +90,34 @@ class Meta:
 
     name = factory.Faker('word')
     form_fields = blocks.ReviewFormFieldsFactory
+
+
+class AbstractRelatedReviewFormFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = AbstractRelatedReviewForm
+        abstract = True
+    form = factory.SubFactory(ReviewFormFactory)
+
+
+class ReviewFundTypeFactory(FundTypeFactory):
+
+    @factory.post_generation
+    def review_forms(self, create, extracted, **kwargs):
+        if create:
+            fields = build_form(kwargs, prefix='form')
+            for _ in self.workflow.stages:
+                # Generate a form based on all defined fields on the model
+                FundReviewFormFactory(
```

Comment:
Looks like this needs to move to the funds model factory :/

The funds admin forms tests are failing